.github/workflows/build-release.yaml

```
@@ -8,6 +8,9 @@ on:
 jobs:
   release:
     runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
```

```suggestion
      contents: write
```

read permission is not enough for creating a tag.